include/xtensor/xcontainer.hpp

```diff
@@ -220,7 +190,7 @@ namespace xt
         using inner_strides_type = typename base_type::inner_strides_type;
         using inner_backstrides_type = typename base_type::inner_backstrides_type;
 
-        void reshape(const shape_type& shape);
+        void reshape(const shape_type& shape, bool force = false);
         void reshape(const shape_type& shape, layout l);
         void reshape(const shape_type& shape, const strides_type& strides);
```

Comment:
Hmm, I tried this, but it requires rewriting almost all constructors for xarray and xtensor.